### DISCUSSION TITLE
可否有近期新增笔记的显示列表功能

### DISCUSSION THREAD

**Discussion by ZeroZap** [obsidianzh/forum#63] [Q&A常见问题]
求助下：obsidian可以通过命令面板默认显示近期编辑笔记，但是可否有显示近期新增笔记的列表功能呢，类似云笔记这样的
![image](https://user-images.githubusercontent.com/58361840/108614455-877ae900-7435-11eb-9d53-0d85fb87351d.png)

**Comment by makerjackie** [comment]
也许你可以通过 window 文件浏览器查看创建时间...
